### ISSUE TITLE
Need ability to include buttons in grids or tables

### ISSUE THREAD

**Issue by apetitjean** [ironmansoftware/universal-dashboard#117]
**Tested with 1.6.0b3**

It would be super cool if we could - instead of a link - insert a button in a row.

For example:
```PowerShell
$PageGridWithLinks1 = New-UDPage -Name "Grid with links" -Content {

    $Data = @(
        [PSCustomObject]@{Animal="Frog";Order="Anura";Article=( New-UDButton -Text 'Click me' -OnClick { Set-UDElement -Id 'aLabel' -Content { "Clicked!" }})}
        [PSCustomObject]@{Animal="Tiger";Order="Carnivora";Article=(New-UDLink -Text "Wikipedia" -Url "https://en.wikipedia.org/wiki/Tiger")}
        [PSCustomObject]@{Animal="Bat";Order="Chiroptera";Article=(New-UDLink -Text "Wikipedia" -Url "https://en.wikipedia.org/wiki/Bat")}
        [PSCustomObject]@{Animal="Fox";Order="Carnivora";Article=(New-UDLink -Text "Wikipedia" -Url "https://en.wikipedia.org/wiki/Fox")}
    )

    New-UDGrid -Title "Animals" -Headers @("Animal", "Order", "Wikipedia") -Properties @("Animal", "Order", "Article") -Endpoint {
        $Data | Out-UDGridData
    }
}
```

I get this error : 
```
An error occurred on this page
 Minified React error #31; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BRefreshInterval%2C%20Callback%2C%20ComponentName%2C%20Error%2C%20JavaScriptId%2C%20AutoRefresh%2C%20JavaScriptPath%2C%20Properties%2C%20Attributes%2C%20ModuleName%2C%20Content%2C%20Events%2C%20Type%2C%20Tag%2C%20Id%2C%20HasCallback%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

**Comment by adamdriscoll** [comment]
@apetitjean You should be able to do it in a table but not a grid yet. I hope to get that implemented in a grid via #44

**Comment by apetitjean** [comment]
Indeed I can insert a button in a Table. Thanks for the tip! ;-)

By the way, do you know how can I adjust the space between the rows? 
I tried just to see what it does, the -Style responsive-table but it doesn't have any effect.

**Comment by fmichaleczek** [comment]
@apetitjean you can add a css to your dashboard and customize ud rendering.

**Comment by apetitjean** [comment]
Thanks @fmichaleczek ! I’m gonna try that.

**Comment by adamdriscoll** [comment]
Implemented in 1.6.0.
